Configuration/AlCa/python/autoAlca.py

```diff
@@ -49,7 +50,7 @@ def buildList(pdList, matrix):
     return stringList
 
 # Update the lists anytime a new PD is added to the matrix
-autoAlca = { 'allForPrompt'         : buildList(['Charmonium', 'Commissioning', 'DoubleMuParked', 'DoubleMuon', 'EGamma', 'HLTPhysics', 'HcalNZS', 'JetHT', 'MET', 'MinimumBias', 'MuOnia', 'MuOniaParked', 'NoBPTX', 'SingleMuon', 'ZeroBias'], AlCaRecoMatrix),
+autoAlca = { 'allForPrompt'         : buildList(['Charmonium', 'Commissioning', 'DoubleMuParked', 'DoubleMuon', 'DoubleMuonLowMass', 'EGamma', 'HLTPhysics', 'HcalNZS', 'JetHT', 'MET', 'MinimumBias', 'MuOnia', 'MuOniaParked', 'NoBPTX', 'SingleMuon', 'ZeroBias'], AlCaRecoMatrix),
```

Comment:
'Charmonium', 'Muonia', 'LowMassNonResonant' datasets are now (https://its.cern.ch/jira/browse/CMSHLT-2232) merged into 'DoubleMuonLowMass': should the removed dataset remain in this list? (Just asking, simply because it is not fully obvious to me how does this work...)

Reply:
Hi Andrea, there is interplay now here with how the RelVals are set up. Now that the tests are still using Run-2 data, which has the above 3 PDs still, it's fine. In the case of the T0 it will just not do anything given that the streamers for this are not available. When we finally have the full physics menu at HLT (IIUC that's in 3rd and 4th week on June), we (or PdmV rather) could modify the RelVals and get rid of these 3 from this list